packages/testing/examples/useBlockMeta.test.tsx

```diff
@@ -1,27 +1,27 @@
 import { expect } from 'chai'
-import { useBlockMeta } from '../src'
-import { renderWeb3Hook, sleep } from '../src/testing'
+import { useBlockMeta } from '@usedapp/core'
+import { renderWeb3Hook, sleep } from '../src'
```

Comment:
I wonder if it should be in 'examples' directory. It seems like a real test of core package.